src/figuresClasses.ts

```diff
@@ -1,11 +1,94 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  color: string;
+
+  private a: number;
+
+  private b: number;
+
+  private c: number;
+
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side lengths must be greater than 0');
+    }
+
+    if (a >= b + c || b >= a + c || c >= a + b) {
+      throw new Error(
+        `The longest side can't be greater than the sum of the other two sides`,
+      );
+    }
+
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
```

Comment:
[CODE KNOWLEDGE] - When you declare a constructor parameter with an access modifier TypeScript implicitly creates a property on the class with the same name as the parameter and assigns the parameter value to that property. You don't need to do it manually. Instead, you can use the access modifier in the constructor parameter list to simplify your code.